### ISSUE TITLE
Request: Change powered ride speeds

### ISSUE THREAD

**Issue by austinjb555** [OpenRCT2/OpenRCT2#1809]
I would like the ability to change ride speeds such as log flume, trams, monorails, etc. Can we get a "ride speed" under the ride's mechanics menu, similar to chairlift speed?

**Comment by Christipher** [comment]
The log flume, Splash Boats, river rapids, etc. are all affected by gravity.  The speed of the water would be the only thing in there, and it might wind up becoming troublesome.

The monorail should be more gradual in its acceleration though each type would have it's own maximum speed as well... let alone the turns also having it's maximum speed (which is represented in Locomotion).  Same goes for trains and trams.

Now as for things Like the Ghost Train and Haunted Mansion rides - those should get that ability and be more controlled (the two examples are the worst offenders of being incorrect, since they act like water rides and are the most prone to crashing by flying off a turn, though less dramatic examples also exist, such as the car ride and Cheshire cat ride).

Now as for Monorail Cycles and Go Karts - those are powered by the guest, and the monorail cycles should also be more like go karts (though at speeds like of a water ride).

There's my two cents on this... sad to say I can't do anything about this myself, but maybe Orca can..

**Comment by austinjb555** [comment]
Yeah, the speed of water (and lift hill speed)for log flume, river rapids, and splash boats would be wonderful to be able to change. I also wanted to change the monorail/train because it goes very slow up hills. Oh well, maybe this can get done at some point in the future.

**Comment by Gymnasiast** [comment]
The uphill speed of the monorail doesn't seem to be terribly realistic anyway, especially for the streamlined trains.
